server/server.ts

```diff
@@ -52,7 +52,8 @@ if (featureSwitches.cspSecurityAudit) {
 		'report-uri /api/csp-audit-report-endpoint',
 		'report-to csp-endpoint',
 		`default-src ${cspDefaultSrcAllowList}`,
-		`default-style 'unsafe-inline'`, // this is unsafe but needed for now for emotion
```

Comment:
no idea where we got `default-style` from 😕

Reply:
my fault I think I read the csp report wrong 😖